BUILDING.md

```diff
@@ -0,0 +1,21 @@
+Building
+--------
+
+You can choose to build this library either with `cmake` or `automake`.
+The pre-requisite for compilations are:
+
+- you should have cloned this repository with `git clone https://github.com/libsigcplusplus/libsigcplusplus.git`
```

Comment:
Most people would get via a release tarball, not via git. The build is generally more difficult directly from git.

Reply:
@murraycu this is generally true but it is good for people who would like to embed `libsigc++` into their software (or for package maintainers) to have build instructions ;)